core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -813,41 +813,56 @@ class Partition(val topicPartition: TopicPartition,
    */
   private def tryCompleteDelayedRequests(): Unit = delayedOperations.checkAndCompleteAll()
 
+  // Visible for testing.
+  private[cluster] def shouldShrinkIsr(): Boolean = {
```

Comment:
nit: to be consistent with the caller below, we can drop the parenthesis.

Reply:
Added parenthesis to call site, since it appears convention to use them. I presume it's more self-documenting as well that we're referencing a function and not a variable.

Reply:
Done.